tsuru/client/auth.go

```diff
@@ -271,6 +274,128 @@ func (c *TeamList) Run(context *cmd.Context, client *cmd.Client) error {
 	return nil
 }
 
+type TeamInfo struct{}
+
+func (c *TeamInfo) Info() *cmd.Info {
+	return &cmd.Info{
+		Name:    "team-info",
+		Usage:   "team-info <team>",
+		Desc:    `Shows information about a specific team.`,
+		MinArgs: 1,
+	}
+}
+
+func (c *TeamInfo) Run(context *cmd.Context, client *cmd.Client) error {
+
+	context.RawOutput()
+	team := context.Args[0]
+	u, err := cmd.GetURL(fmt.Sprintf("/teams/%v", team))
```

Comment:
You should use `GetURLVersion("1.4", fmt.Sprintf("/teams/%s", team))`, because this endpoint isn't available before tsuru version 1.4